### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by rondale-sc**

### CODE REVIEW

**Comment by rwjblue** [comment]
[rendered](https://github.com/rondale-sc/rfcs/blob/rehydration-rfc/text/0000-rehydration-by-default.md)

File: text/0000-rehydration-by-default.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+content of the block.
+
+<small>(from: https://github.com/glimmerjs/glimmer-vm/commit/316805b9175e01698120b9566ec51c88d075026a)<small>
+
+Currently Fastboot rehydration is available behind a (rather lengthy) environment variable: `EXPERIMENTAL_RENDER_MODE_SERIALIZE=true`.  The Goal of this RFC is to enable this by default and explore the work necessary to accomplish that.
+
+## Motivation
+
+Much of the work done in FastBoot today is duplicated because rehydration is not enabled.  In fact, the current implementation of `ember-cli-fastboot` has an initializer ([here](https://github.com/ember-fastboot/ember-cli-fastboot/blob/master/addon/instance-initializers/clear-double-boot.js)) called `clear-double-boot` which removes all of the work done in SSR land so the Glimmer VM has a fresh start to begin creating the DOM. With Rehydration the work done on the server is not thrown away and instead re-used, so removing the `clear-double-boot` should result in faster websites.
+
```
Comment (sdhull) [inline text/0000-rehydration-by-default.md:44]: "faster websites" is a good motivation

I'd also mention that a downside of `clear-double-boot` is that if a user starts filling out a form before the app has completed the in-browser boot, their entries are clobbered by the `double-boot` process. This is relatively "uncommon" however we've seen it happen on cell phones when assets take an inexplicably long time to load.

Replacing the entire dom with a newly-generated dom is potentially like having the rug pulled out from under you. So "motivation" here is "not pulling the rug out from under users."

Comment (rondale-sc) [reply text/0000-rehydration-by-default.md:44]: I've updated a section around `scenarios` to point this out

**Comment by sdhull** [comment]
Is the eventual target for this RFC glimmer-vm or ember or ...?

**Comment by rondale-sc** [comment]
@sdhull The eventual target of this is fastboot, and ember-cli-fastboot projects.

**Comment by rwjblue** [comment]
@sdhull - I think it will ultimately be PR'ed against emberjs/rfcs.

**Pull request closed without merging**